### ISSUE TITLE
aws_acm_certificate - Tagging is not permitted on re-import.

### ISSUE THREAD

**Issue by frimik** [terraform-providers/terraform-provider-aws#15055]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

```
Terraform v0.13.2
+ provider registry.terraform.io/-/aws v3.5.0
+ provider registry.terraform.io/-/local v1.4.0
+ provider registry.terraform.io/hashicorp/aws v3.5.0
+ provider registry.terraform.io/hashicorp/local v1.4.0
+ provider registry.terraform.io/hashicorp/null v2.1.2
+ provider registry.terraform.io/hashicorp/template v2.1.2
+ provider registry.terraform.io/hashicorp/tls v2.2.0
```

### Affected Resource(s)

* aws_acm_certificate

### Terraform Configuration Files

```hcl
resource aws_acm_certificate imported {
  provider = aws.regional

  private_key       = data.local_file.key.content
  certificate_body  = data.local_file.crt.content
  certificate_chain = data.local_file.chain.content

  lifecycle {
    create_before_destroy = true
  }

  tags = {
    Name            = "domain_name"
    Terraform       = "some-infra"
    TerraformModule = "some-modules/aws-acm-certificate-import"
  }

}
```

### Expected Behavior

Update certificate contents but retain tags as-is or perform two API requests separately.

### Actual Behavior

Certificate failed to update in place. Error message:

```
Error: Error updating certificate: ValidationException: Tagging is not permitted on re-import.
```

### Steps to Reproduce

1. `terraform apply`
2. update certificate file contents (from disk or strings)
3. `terraform apply` again
4. observe the error message "Tagging is not permitted on re-import."

### References

https://docs.aws.amazon.com/acm/latest/userguide/import-reimport.html states that:

> You cannot apply resource tags when reimporting a certificate.

**Comment by bflad** [comment]
Fix submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/15060

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.7.0 of the Terraform AWS Provider, later this week. 👍
